problems/arrayMethods.js

```diff
@@ -91,15 +152,29 @@ const targetCount = () => {};
  * @returns {Object} A mapping of elements to their occurrence count.
  */
 
-const counterObject = () => {};
-
+const counterObject = (items) => {
+  let objCount = {}
+  items.forEach((item) => {
+    if (objCount[item] = objCount[item]? objCount[item] + 1 : 1) {
+    }
```

Comment:
This does not belong in a if statement, you can leave the if statement out because you are already evaluating   truthy or falsy with the ternary operators